internal/config/schemas.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+const defaultSchemasPath = "cue/schemas"
+
+type Schemas struct {
+	Path string `yaml:"path,omitempty"`
```

Comment:
do you expect to add more configuration there ? It looks heavy for the moment to add a new struct for just one field

Reply:
Yes indeed, not relevant for the moment